cmake/SetupTestTargets.cmake

```diff
@@ -11,7 +11,7 @@ IF(MUQ_USE_GTEST)
     set(all_gtest_sources modules/RunTests.cpp)
     set(all_gtest_sources_parallel modules/RunParallelTests.cpp)
     set(all_compiled_libraries )
-    
+
```

Comment:
let's avoid these changes for extra spaces/lines that only pollute the PR , similarly below